install.sh

```diff
@@ -87,11 +87,19 @@ quote() {
   done
 }
 
-# Add indentation and trailing slash to quoted args
+# Add indentation and trailing slash to quoted args except last one
+# Also don't add trailing slash to command if no args are given
 quote_indent() {
-  printf ' \\\n'
+  if [ $# -ge 1 ]; then
+    printf ' \\\n'
+  fi
+  _arg_count=1
```

Comment:
Can you move `_arg_count=1` to top (only for style) and add an empty space:
```sh
_arg_count=1

if [ $# -ge 1 ]; then
    printf ' \\\n'
fi

...
```

Reply:
Done